packages/docs/getting-started/running-a-full-node-in-rc1.md

```diff
@@ -0,0 +1,111 @@
+# Running a Full Node
+
+This section explains how to get a full node running on the [Release Candidate 1 Network](rc1.md), using a Docker image that was built for this purpose.
+
+Full nodes play a special purpose in the Celo ecosystem, acting as a bridge between the mobile wallets \(running as light clients\) and the validator nodes. To make sure that full nodes are rewarded for this service, the Celo protocol includes [full node incentives](../celo-codebase/protocol/transactions/full-node-incentives.md).
+
+When a light client sends transactions, they will include specify a gateway fee to be paid to the node that gossips transactions to the other full nodes and validators.
```

Comment:
@asaj do we know when full node incentives will be enabled?

Reply:
This is not yet enabled because the mechanism for negotiating a gateway fee is not yet built. Currently, light clients are set to send a gateway fee of 0 and full nodes are set to accept a minimum gatewayfee of 0

Reply:
Added a warning to explain this.